app/controllers/application_controller.rb

```diff
@@ -1,2 +1,20 @@
 class ApplicationController < ActionController::Base
+
+  before_action :authenticate_member!
+
+  def authenticate_member!
+    if current_member == nil
+      redirect_to("/sign_in", { :alert => "You must be signed in." })
+    end
+  end
+
+  helper_method :current_member
+
+  def current_member
```

Comment:
For simplicity, and since this method is just training wheels anyway (and thus doesn't need to be performant/best practice), what do you think about dropping the helper method altogether?

Instead, we'll just set an instance variable `@current_member` in a `before_action` and teach them to use it directly. This should be less magical to students.

See this workspace for an example:

https://codio.com/dstudent43/photogram-signin

Thoughts?

Reply:
I agreem the `before_action` is most likely more clear than the `helper_method`. I'll make the change.